source/custom/nirfmxinstr_restricted_service.custom.cpp

```diff
@@ -1,2 +1,12 @@
+#include <nirfmxinstr_restricted/nirfmxinstr_restricted_service.h>
+
 namespace nirfmxinstr_restricted_grpc {
-}  // namespace nirfmxinstr_restricted_grpc
\ No newline at end of file
+
+::grpc::Status NiRFmxInstrRestrictedService::ConvertApiErrorStatusForNiRFmxInstrHandle(google::protobuf::int32 status, niRFmxInstrHandle instrumentHandle)
+{
+    std::string description(nidevice_grpc::kMaxGrpcErrorDescriptionSize, '\0');
+    // TODO: How do we get access to a library that can return a useful string?
```

Comment:
`nirfmxinstr_library` and `nirfmxinstr_restricted_library` are C++ wrappers for different subsets of the same DLL. Perhaps you could add `GetError` to `nirfmxinstr_restricted`'s metadata and set `'codegen_method': 'private'` to keep it out of `nirfmxinstr_restricted.proto`.

Reply:
Since typing that, I had thought of pretty much the same thing. After this PR gets submitted, though. I've already had to fetch/merge 3 times and it's getting to be not-fun.

Reply:
[Task 2107350](https://ni.visualstudio.com/DevCentral/_workitems/edit/2107350): Ensure RF restricted APIs have access to "error message" function.